app/helpers/search_results_helper.rb

```diff
@@ -1,4 +1,25 @@
 module SearchResultsHelper
+  def descriptions_with_other_handling(commodity)
+    ancestor_descriptions = commodity.ancestor_descriptions.reverse
+    return sanitize commodity.description.to_s unless commodity.description.to_s.match(/^other/i)
+
```

Comment:
this guard clause does not use ` ancestor_descriptions `, it could be move one line up:
```
    return sanitize commodity.description.to_s unless commodity.description.to_s.match(/^other/i)
    ancestor_descriptions = commodity.ancestor_descriptions.reverse
```